.github/workflows/build.yml

```diff
@@ -1,7 +1,6 @@
 name: Continuous Integration Checks
 
-on:
-  push:
+on: [push, pull_request]
```

Comment:
ah, I had mistakenly assumed `push` would encompass `pull_request`..

Reply:
Yep, I just came back to master to fix that as I realised it was needed for fork PRs. Well done @TheBlueMatt :)